tsconfig.json

```diff
@@ -8,8 +8,14 @@
     "noEmit": true,
     "skipDefaultLibCheck": true,
     "strict": true,
-    "target": "es5"
+    "target": "es5",
+    "types": []
   },
   "include": ["packages", "scripts", "site", "test", "playgrounds"],
-  "exclude": ["node_modules", "packages/*/types/test/*"]
+  "exclude": [
+    "node_modules",
+    "packages/*/types/test/*",
+    "packages/*/types/test*",
```

Comment:
what the second one is doing here? 🤔

Reply:
Not all of the type tests are in a test directory, maybe the first one is unnecessary, idk how TS's globs work